include/libnuraft/peer.hxx

```diff
@@ -498,6 +502,8 @@ private:
      */
     std::atomic<bool> abandoned_;
 
+    std::atomic<bool> lost_by_leader_ {false};
```

Comment:
Please add comment about this member variable.

Also, instead of brace initializer here, please put the default value into constructor, to be aligned with others.